frontend/src/components/Home.vue

```diff
@@ -141,186 +178,122 @@ export default {
   name: 'Home',
   data() {
     return {
-      menu: [
-        { title: 'Welcome',
-          text: '<b>Metabolic Atlas</b> integrates open source genome-scale metabolic models (GEMs) of human and yeast for easy browsing and analysis. It also contains many more GEMs constructed by our organization.<br><br>Detailed biochemical information is provided for individual model components, such as reactions, metabolites, and genes. These components are also associated with standard identifiers, facilitating integration with external databases, such as the Human Protein Atlas. <br><br><a href="https://doi.org/10.1126/scisignal.aaz1482" target="_blank">Robinson, J., et al, 2020. <i>An atlas of human metabolism</i>. Sci. Signal. 13, eaaz1482 </a>',
-          img: require('../assets/explorer.jpg'),
-          cardLink: 'Explore a GEM',
-          route: { name: 'explorer' },
-          icon: 'home' },
+      news: {
+        title: "What's new",
+        text: 'Here goes all the news, which ideally should link to the news page under About.',
```

Comment:
This looks like a placeholder text. Is there any Jira task in the current sprint to replace this?

Reply:
MA-664 should cover it.

Reply:
For now, resolved in 820e49d